index.html

```diff
@@ -133,6 +133,7 @@
 				<li><a href="#achievements" class="smoothScroll">My Achievements</a></li>
 				<!-- <li><a href="#price" class="smoothScroll">Prices</a></li> -->
 				<li><a href="#testimonial" class="smoothScroll">Reviews</a></li>
+				>div><My Career>div>

```

Comment:
The content within the div tags should be wrapped in proper HTML tags. Consider using <div>My Career</div> instead of >div><My Career>div>.